src/Owin.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -110,6 +124,8 @@ public static class Metadata {
             public const string SubjectTypesSupported = "subject_types_supported";
             public const string TokenEndpoint = "token_endpoint";
             public const string UserinfoEndpoint = "userinfo_endpoint";
+            //Added by Introspection (but not in Spec?)
+            public const string IntrospectionEndpoint = "introspection_endpoint";
```

Comment:
@MonkeyJamboree it looks like we made the right choice, as it's named `introspection_endpoint` in the whole new OAuth2 discovery draft: http://tools.ietf.org/html/draft-jones-oauth-discovery-00#section-3 :clap: